### PR TITLE
Frontend component tests

### PR DESCRIPTION
**Pull request by israelgonzalezb**
# Description
Added tests across the frontend

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [x] Tests validated

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by AlyssaSaez — APPROVED** [review]
Thank you for your service

File: src/components/FoodItem/index.js [hunk 1]
```diff
@@ -2,48 +2,37 @@ import React from "react";
...
-      searchTerm: ""
-    };
-  }
+
   render() {
     const { path } = this.props.match;
     return (
-      <Container height={this.props.height} fluid>
+      <Container height={this.props.height}>
         <Route
```
Comment (dave-vazquez) [inline src/components/FoodItem/index.js]: there's some changes to code here that aren't tests?

**Review by dave-vazquez — CHANGES_REQUESTED** [review]
saw a few changes here to files that weren't tests, just want to be cautious before merging in

File: src/setUpTests.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+function _bigMockStore_(initialState = {}, middlewares = []) {
+  // Global variables should have a _ at the beginning and at the end
+  const mockStore = configureStore([...middlewares, thunk]);
+  return mockStore(initialState);
+}
+
+// Takes your mounted wrapper, an array of components to look for, and whether you want to console log some details
+function _doesThisRender_(wrapper, components, logger = false) {
+  logger
```
Comment (dave-vazquez) [inline src/setUpTests.js:16]: I added this function back cause it doubled our passing tests, can always be removed if necessary

File: src/setUpTests.js [hunk 3]
```diff
@@ -1,15 +0,0 @@
-import configureStore from "redux-mock-store";
```
Comment (dave-vazquez) [inline src/setUpTests.js:1]: delete one file, another one appears... delete that file, the first one appears 👻 👻 👻

**Comment by dave-vazquez** [comment]
Okey-doke, this one looks ready to go 🎉

**Review by dave-vazquez — APPROVED** [review]

**Pull request merged**